### DISCUSSION TITLE
Support for SvelteKit

### DISCUSSION THREAD

**Discussion by yuxxeun** [heroui-inc/heroui#646] [Ideas]
enable integration between `Next UI` and `SvelteKit` (?), it will make a cool combination.

**Comment by jrgarciadev** [comment]
Hey @yuxxeun at the moment we are only focused on React and its frameworks/toolings, but we hope to have support for other frameworks in the future 🙌🏼

**Reply by yuxxeun** [reply]
glad to hear that anyway

**Reply by tonprince** [reply]
The combination between Next UI and SvelteKit would be fantastic, I agree with @yuxxeun

**Comment by 0x090909** [comment]
UP UP I also would like a lot to see that come to life

**Comment by AxelBriche** [comment]
Up

**Comment by tlherysr** [comment]
UP

**Comment by ryneex** [comment]
UP

**Comment by Yuutakasan** [comment]
UP

**Comment by andrereliquias** [comment]
UP

**Comment by 1Kanan1** [comment]
It's been 3 years. Even though I hope they'll integrate it into Svelte, I still believe this won't happen anytime soon.

**Comment by wingkwong** [comment]
Just an update. This has been planned in v3.

**Reply by 0x090909** [reply]
when is v3 expected to be released

**Reply by wingkwong** [reply]
currently it's still on alpha version. gotta be react first. I don't have the exact timeline for svelte part.

**Reply by E021ntox** [reply]
Any updates?

**Comment by julien-blanchon** [comment]
Up

**Comment by ddtamn** [comment]
Hi 👋 — I tried using `@heroui/styles` in Svelte based on the “framework-agnostic styles” docs, and it works quite well for building native components with the same variants (no React runtime).

Playground:
https://svelte.dev/playground/c0227b7e25cb45ada7f8ce3d80d0e502

Just sharing as an example 🙌
